skops/cli/_convert.py

```diff
@@ -0,0 +1,109 @@
+from __future__ import annotations
+
+import argparse
+import logging
+import os
+import pathlib
+import pickle
+from typing import Optional
+
+from skops.cli._utils import get_log_level
+from skops.io import dumps, get_untrusted_types
+
+
+def _convert_file(input_file: os.PathLike, output_file: os.PathLike):
+    """
+    Function that is called by ``skops convert`` entrypoint.
+
+    Loads a pickle model from the input path, converts to skops format, and saves to
+    output file.
+
+    Parameters
+    ----------
+    input_file : os.PathLike
+        Path of input .pkl model to load.
+
+    output_file : os.PathLike
+        Path to save .skops model to.
+
+    """
+    model_name = pathlib.Path(input_file).stem
+
+    logging.debug(f"Converting {model_name}")
+
+    with open(input_file, "rb") as f:
+        obj = pickle.load(f)
+    skops_dump = dumps(obj)
+
+    untrusted_types = get_untrusted_types(data=skops_dump)
+
+    if not untrusted_types:
+        logging.info(f"No unknown types found in {model_name}.")
+    else:
+        untrusted_str = ", ".join(untrusted_types)
+
+        logging.warning(
+            "Unknown Types Detected! "
+            f"While converting {model_name}, "
+            "the following unknown types were found: "
+            f"{untrusted_str}. "
+            f"When loading {output_file}, add 'trusted=True' to the skops.load call. "
```

Comment:
I think from a users' perspective, it might be nicer to give them the easiest fix possible. 

When they're converting this file, and want to not deal, the log line above would've told them all of the untrusted types anyway, so if they wanted to go that route, they could just add those.

We could add a line along the lines of:
`..., or set each of the unknown types listed previously to "trusted".`